dexmaker-tests/src/androidTest/java/com/android/dx/stock/ProxyBuilderTest.java

```diff
@@ -685,18 +699,18 @@ public void testClassNotCachedWithDifferentParentClassLoaders() throws Exception
 
         assertTrue(a.getClass() != b.getClass());
     }
-    
+
     public void testAbstractClassWithUndeclaredInterfaceMethod() throws Throwable {
         DeclaresInterface declaresInterface = proxyFor(DeclaresInterface.class)
                 .build();
         assertEquals("fake result", declaresInterface.call());
         try {
             ProxyBuilder.callSuper(declaresInterface, Callable.class.getMethod("call"));
             fail();
-        } catch (AbstractMethodError expected) {
+        } catch (IncompatibleClassChangeError expected) {
```

Comment:
`AbstractMethodError` extends `IncompatibleClassChangeError`

Dalvik throws `AbstractMethodError` and ART throws `IncompatibleClassChangeError`, so catching `IncompatibleClassChangeError` works for both